Modules/ContentKit/Sources/Content/Categories/Category+Curation.swift

```diff
@@ -0,0 +1,165 @@
+// Leka - iOS Monorepo
+// Copyright APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+import LocalizationKit
+import SwiftUI
+import UtilsKit
+import Yams
+
+// MARK: - CurationItemModel
+
+// swiftlint:disable nesting
+
+public struct CurationItemModel {
+    public let uuid: String
+    public var contentType: CurationType = .curation
+}
+
+// MARK: - CategoryCuration
+
+public struct CategoryCuration: Identifiable, CategoryProtocol {
+    // MARK: Lifecycle
+
+    public init?(id: String) {
+        if let curation = ContentKit.allCurations.first(where: { $0.id == id }) {
+            self = curation
+        } else {
+            return nil
+        }
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        self.uuid = try container.decode(String.self, forKey: .uuid)
+        self.icon = try container.decode(String.self, forKey: .icon)
+        let hexColor = try container.decode(UInt.self, forKey: .color)
+        self.color = Color(hex: hexColor)
+        self.l10n = try container.decode([Category.LocalizedDetails].self, forKey: .l10n)
+        self.sections = try container.decode([CategoryCuration.Section].self, forKey: .content)
+    }
+
+    // MARK: Public
+
+    public let uuid: String
+    public var icon: String
+    public var color: Color
+    public var l10n: [Category.LocalizedDetails]
+    public var sections: [CategoryCuration.Section]
+    public var contentType: CurationType = .curation
```

Comment:
depending on the answer to my previous question, use ContentType and keep the variable name or use CurationType but change the variable name to `var curationType`